src/contexts/toast/ToastContext.tsx

```diff
@@ -34,19 +35,23 @@ type ToastType = {
   message: string;
   onDismiss: DismissToastHandler;
   autoClose: boolean;
+  variant: 'success' | 'error';
```

Comment:
nit: looks like this is repeated in Line 11

could do `type ToastVariant = 'success' | 'error'` and use that